src/components/Checkbox/Checkbox.js

```diff
@@ -43,6 +44,25 @@ export default class Checkbox extends React.Component {
           }
         });
 
+    Array(11).fill()
```

Comment:
Let's avoid the double iteration.
We need to populate `checkboxesToRender` or `checkboxesToRenderGrid`not both.
We could unify the two by using the same array `checkboxesToRender`

Reply:
double iteration removed